conf/messages

```diff
@@ -117,5 +117,21 @@ classOfBeneficiary.entityStart.error.invalid = Enter a real date when the class
 classOfBeneficiary.entityStart.error.future = The date the class of beneficiaries was added to the trust must be today or in the past
 classOfBeneficiary.entityStart.error.past = The date the class of beneficiaries was added to the trust cannot be before the start date of the trust
 
-classOfBeneficiary.checkDetails.title = Check class of beneficiaries details
-classOfBeneficiary.checkDetails.heading = Check class of beneficiaries details
\ No newline at end of file
+classOfBeneficiary.checkDetails.title = Check class of beneficiary details
+classOfBeneficiary.checkDetails.heading = Check class of beneficiary details
+
+removeClassOfBeneficiary.title = Are you sure you want to remove {0}?
+removeClassOfBeneficiary.heading = Are you sure you want to remove {0}?
+removeClassOfBeneficiary.default = the beneficiary
+removeClassOfBeneficiary.error.required = Select yes if you want to remove the beneficiary
+
+trustee.whenRemoved.checkYourAnswersLabel = When was {0} removed as a beneficiary?
+classOfBeneficiary.whenRemoved.title = When was the trustee removed as a beneficiary?
```

Comment:
This says trustee.

Reply:
Oops, this key should have been deleted, it's not used. Removed Now.